core/shared/src/test/scala/fs2/HashSuite.scala

```diff
@@ -55,7 +55,7 @@ class HashSuite extends Fs2Suite with HashSuitePlatform with TestPlatform {
   }
 
   test("empty input") {
-    Stream.empty[IO].through(sha1).compile.count.assertEquals(20L)
+    Stream.empty.covary[IO].through(sha1).compile.count.assertEquals(20L)
```

Comment:
On adding the pretty printer implicit conversion, `Stream.empty` resolved to the `val empty` instead of `def empty[A]` on the monoidK instance. I'm at a loss as to why.

Reply:
See [the comment below](https://github.com/typelevel/fs2/pull/3242#discussion_r1241270269)